### ISSUE TITLE
Crash due to spaces in username on v0.5

### ISSUE THREAD

**Issue by dirtling** [bamhm182/FinTrinity#16]
v0.4 had the same problem, didn't try v0.3 but v0.2 worked fine.

**Comment by bamhm182** [comment]
Fails with the following:

```C:\Users\User Name\...\pscimg-extract...
'C:\Users\User' is not recognized as an internal or external command...
```
I need to wrap paths in quotes.

**Comment by bamhm182** [comment]
Resolved in v0.6. I had to improve how the commands were being run.
